app/src/main/java/ch/epfl/polychef/fragments/PostRecipeFragment.java

```diff
@@ -47,17 +51,25 @@
     private final int titleMaxChar = 80;
     private final int titleMinChar = 3;
     private final int maxPersNb = 100;
+    private final int maxInstructions = 20;
```

Comment:
Est-ce qu'on essaierai de mettre toutes les constantes en majuscule ou ici c'est overkill ?

Reply:
Ca ne sera pas utile et ça pose pas de souçis à codeclimate donc overkill non?

Reply:
C'est une convention de mettre les constantes en majuscule et en plus on les voit mieux, c'est plus cohérent et mieux repérable, je pense qu'on doit mettre ces variable en majuscule